docs/site/migration/auth/example.md

```diff
@@ -446,143 +448,64 @@ Try the 'team-member' role:
 
 ### Model Creation
 
-Create User Model
+Import model `User`, `Project`, `Team` by the migration CLI:
```

Comment:
'Import model `User`, `Project`, `Team` from the LB3 app by the migration CLI:' ?